src/Query/LaravelReadQuery.php

```diff
@@ -100,6 +103,7 @@ public function getResourceSet(
                 foreach ($order->getSubPathSegments() as $subOrder) {
                     $subName = $subOrder->getName();
                     $subName = (self::PK == $subName) ? $keyName : $subName;
+                    $subName = $tableName.'.'.$subName;
```

Comment:
Prepending the table name here to try to ensure that the column being sorted on is unambiguous.

Joining two conventional tables (which each have id as their PK columns) and then ordering by id on one of the tables would break the join with an ambiguous-column error - which table's id column are you ordering on?